src/main/java/ui/UiRenderer.java

```diff
@@ -199,10 +199,10 @@ private static void storeWrappedLines(LocalDate startOfWeek, TaskManager taskMan
 
             for (Task task : dayTasks) {
                 String taskDescription = task.getName();
-                String displayString = 
-                        (dayTasks.indexOf(task) + 1) + "." + 
-                        task.getDisplayFormat() +
-                        taskDescription;
+                String displayString =
+                        (dayTasks.indexOf(task) + 1) + "." +
+                                task.getDisplayFormat() +
+                                taskDescription;
```

Comment:
Would the original formatting be following the coding standard better? An example from https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html shows the folllowing example:

```
alpha = (aLongBooleanExpression)
        ? beta
        : gamma;
```

Reply:
Nice catch there! I had missed it since it had passed the checkstyle tests.